src/bin/pgcopydb/schema.c

```diff
@@ -1429,86 +1432,6 @@ struct FilteringQueries listSourceTablesSQL[] = {
 };
 
 
-/*
- * schema_list_table fetches information for a given table.
- */
```

Comment:
Why remove that one? is it not used at all anymore maybe, because we don't have `--schema-name ... --table-name ...`, or even without that removal, now that we query internal catalogs at `pgcopydb list` time, we might not need that one anymore, right?

If that's the case, please consider opening a separate PR that only removes that one. Like, everything works as it used to, just with less code thanks to a nice cleanup.